src/services/utilities.ts

```diff
@@ -2154,6 +2155,17 @@ export function isStringOrRegularExpressionOrTemplateLiteral(kind: SyntaxKind):
     return false;
 }
 
+export const isStringAndEmptyAnonymousObjectIntersection = (type: Type) => {
+    if (!type.isIntersection()) {
+        return false;
+    }
+
+    const { types, checker } = type;
+    return types.length === 2
+        && (((types[0].flags & TypeFlags.String) && checker.isEmptyAnonymousObjectType(types[1]))
+        || ((types[1].flags & TypeFlags.String) && checker.isEmptyAnonymousObjectType(types[0])));

```

Comment:
Aaaactually, only `string & {}` is recognized as a special form and preserved 😅. The second check will always be false.

<img width="264" alt="image" src="https://user-images.githubusercontent.com/3277153/216691985-e3a0758a-baff-49de-8bbb-b86e7c00503b.png">

Reply:
Wow missed it somehow, hopefully thats easy to fix 😅